### PR TITLE
handle bad filenames

### PR DESCRIPTION
**Pull request by psi29a**
This should take care of:
https://github.com/Mindwerks/worldengine/issues/113

### CODE REVIEW

**Comment by ftomassetti** [comment]
Good idea updating the CHANGELOG, we should use it more

File: MANIFEST.in [hunk 1]
```diff
@@ -1,4 +1,5 @@
 include *.md
+include *.scm
```
Comment (ftomassetti) [inline MANIFEST.in]: what are scm files?

File: worldengine/cli/main.py [hunk 2]
```diff
@@ -407,12 +407,19 @@ def main():
 
     generation_operation = (operation == 'world') or (operation == 'plates')
 
-    produce_grayscale_heightmap = args.grayscale_heightmap
-    if produce_grayscale_heightmap and not generation_operation:
+    if args.grayscale_heightmap and args.grayscale_heightmap.rfind('.') == -1:
+        print("Invalid filename `{}`, you must specify a valid image extension.".format(
+            args.grayscale_heightmap))
+        sys.exit(-1)
```
Comment (ftomassetti) [inline worldengine/cli/main.py]: Should we start using meaningful error codes? Or perhaps adding just a constant for GENERIC_EXIT_CODE?
Just thinking, probably not the useful

**Comment by ftomassetti** [comment]
Really minor comments: merge it!

**Comment by ftomassetti** [comment]
Fix #113

File: MANIFEST.in (hunk 1 above)
Comment (psi29a) [inline MANIFEST.in]: The gimp file that was merged in... for biomes and stuff.

Comment (ftomassetti) [inline MANIFEST.in]: oh, right!

File: worldengine/cli/main.py (hunk 2 above)
Comment (psi29a) [inline worldengine/cli/main.py]: Honestly... I want to get back into the splitting of client code and library code. The library will raise an error, and it is the job of the client code to catch, display the error and just exit.

Python doesn't have constants, but I know what you mean. Right now, it is good enough to give a non zero back that something bad happened.

**Comment by ftomassetti** [comment]
Looks good to have a fixed filename for grayscale and riverrs when running the console app. Should we have more flexibility when we will have an independent library?

Looks good to me, merge it!

**Comment by psi29a** [comment]
We can always be more flexible, but with the CLI always prefixes the seed number. In theory, when used as a library... it is up to the developer to send the filename to the function. So in the GUI for example, we can leave it up to the user to specify the filename and format for output.

**Pull request merged**